tests/test_api.py

```diff
@@ -0,0 +1 @@
+ANBOXCLOUD_INVITATION_CODE = "3GU7UA"
```

Comment:
is it a demo one, or a public invitation code?

Reply:
This is an invitation code the Anbox team gave us that can be used with `@canonical.com` email addresses only.

So it doesn't need to be secret, and can be used for us to test out the demo.

Reply:
ok.